meteor-app/imports/api/orders/orderedParts.js

```diff
@@ -0,0 +1,36 @@
+import { Mongo } from 'meteor/mongo'
+import SimpleSchema from 'simpl-schema'
+
+import { REGEX_ID } from '../schema'
+
+export const orderedParts = new SimpleSchema({
```

Comment:
Please move the sub-schema definition into the schema.js file. It can still be exported.